.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.compile.nullAnalysis.mode": "automatic"
```

Comment:
dont call `git add .` call `git status` and add them one by one or call `git add -u`